docs/source/raft_configuration.md

```diff
@@ -320,7 +320,7 @@ monitor:
 
 ## Troubleshooting
 
-* The more stress you put on your nodes, the more you might have to change certain
+1. The more stress you put on your nodes, the more you might have to change certain
 parameters. As with any system, computer or mechanical, stress can lead to a drag
 in performance. As we noted in the conceptual documentation, leader elections in
 Raft are triggered when follower nodes do not receive either a "heartbeat"
```

Comment:
`either a "heartbeat" messages or an "append" message`

I think that should be `a "heartbeat" message or an "append" message`